src/game.cpp

```diff
@@ -640,11 +640,6 @@ void game::start_game(std::string worldname)
     u.set_highest_cat_level();
     //Calc mutation drench protection stats
     u.drench_mut_calc();
-    if (scen->has_flag("FIRE_START")){
-            m.add_field(u.pos().x + 5, u.pos().y + 3, field_from_ident("fd_fire"), 3 );
-            m.add_field(u.pos().x + 7, u.pos().y + 6, field_from_ident("fd_fire"), 3 );
-            m.add_field(u.pos().x + 3, u.pos().y + 4, field_from_ident("fd_fire"), 3 );
```

Comment:
These offsets are kinda important so the PC doesn't spawn in with a Raging Fire adjacent.  We had that in the initial mergetest version and such starts are unplayable.

(Flashpoint: surrounded by fire on turn 2, On Fire! at turn 3, hoping you saved that char as a template on turn 4.)